filebeat/filebeat.full.yml

```diff
@@ -516,16 +516,15 @@ filebeat.prospectors:
 #
 #processors:
 #- add_docker_metadata:
-#    host: "unix:///var/run/docker.sock"
```

Comment:
IMO we can leave the `host` here, and maybe the `ssl` section as well, since i think it's common to have to change those, right?